lib/SyTest/Federation/Server.pm

```diff
@@ -243,6 +243,25 @@ sub on_request_federation_v1_query_directory
    } );
 }
 
+sub on_request_federation_v1_event
```

Comment:
It is invoked, by the dispatching logic at
  https://github.com/matrix-org/sytest/blob/32b6377f9583446d6c518ab8b11dd12ff33367c5/lib/SyTest/Federation/Server.pm#L188